libraries/default.rb

```diff
@@ -38,5 +38,9 @@ def format_kernel_parameters  # rubocop:disable all
 
       rendered.each { |r| r.prepend('    ') }.join(",\n")
     end
+
+    def format_ssl_versions
+      Array(node['rabbitmq']['ssl_versions']).map { |n| "'#{n}'" }.join(',')
```

Comment:
@michaelklishin, I don't think it is proper to set the default value of ['rabbitmq']['ssl_versions'] to be nil, for:
1) In the rabbitmq document, you don't bother to add a line of
    {ssl, [{versions, ['tlsv1.2', 'tlsv1.1', tlsv1]}]},
    to the /etc/rabbitmq.config to enable the default SSL versions rabbitmq-server supported. It is needed when  
    you want to customize the versions, most of time to limit the set of versions.
    From this perspective, nil just work well, means void or nothiing, right?
2), If we merely change the default value to be a empty array in the attribute file: 
     default['rabbitmq']['ssl_versions'] = []
     I am afraid some code logic will be affected:
     format_ssl_versions if node['rabbitmq']['ssl_versions']
     node['rabbitmq']['ssl_versions'] evaluates to true, thus
     format_ssl_versions returns an empty string '', but unfortunately, '' is evaluated to true as well, so
     finally, we shall end up with the following line in the /etc/rabbitmq.config file
     {ssl, [{versions, []}]},
     which I don't think is a rational result we both want to get, since it is not what the rabbitmq document tells.
3) If we do insist on using [] as the default value, and also follow the rabbitmq document, we have change several  
    checks in the current patch, e.g:
    :ssl_versions => (format_ssl_versions if node['rabbitmq']['ssl_versions'].length != 0)
    But I believe you will also consider it to be less obvious than:
    :ssl_versions => (format_ssl_versions if node['rabbitmq']['ssl_versions'])